CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
+[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v7.0.0-beta.0...v6.0.0-beta.7)
+
+### :rocket: Enhancements
+
+- Fix for border-radius being too large for 3-line entries like in Mail [\#3136](https://github.com/nextcloud/nextcloud-vue/pull/3136) ([jancborchardt](https://github.com/jancborchardt))
+- Add related resources panel component [\#3081](https://github.com/nextcloud/nextcloud-vue/pull/3081) ([Pytal](https://github.com/Pytal))
+- Add a native datetime picker [\#3063](https://github.com/nextcloud/nextcloud-vue/pull/3063) ([JuliaKirschenheuter](https://github.com/JuliaKirschenheuter))
```

Comment:
Non of them is in stable6, I think.
The changelog generator apparently took the wrong branch.

Reply:
did i do smth wrong, or it is ok if i just remove these prs and release it?